src/model/formulations/ArrCon2000/ramp.jl

```diff
@@ -2,6 +2,26 @@
 # Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
 # Released under the modified BSD license. See COPYING.md for more details.
 
+"""
+    _add_ramp_eqs!
```

Comment:
It's conventional to include the signature of the function in the docs, not just the name.

Reply:
Got it. I think I understood "simplified form" from https://docs.julialang.org/en/v1/manual/documentation/ as not including the arguments.

> 1. Always show the signature of a function at the top of the documentation, with a four-space indent so that it is printed as Julia code.
> This can be identical to the signature present in the Julia code (like mean(x::AbstractArray)), or a simplified form.

Reply:
The reason I don't like doing it is that it is incredibly easy and frequent to forget to change the signature in the documentation after adding/changing a function argument, and then the documentation goes out of sync.

Reply:
I agree it's very inconvenient and error-prone. This should really be generated by the documentation tool.

Perhaps the best approach is to not add docstrings to these internal functions, since the users are not supposed to know about them anyway, and since they will not be included in the API docs.

Reply:
I think we need to add them, because it is very important for us as developers. It is hard to keep track of which formulations are using which variables.

Reply:
The workaround I had in my fork of the code was a separate struct that listed all the symbols that are needed for each method. Then I was able to query "what are all the variables in formulation X".